src/simulation/m_kernel_functions.fpp

```diff
@@ -0,0 +1,501 @@
+!>
+!! @file m_kernel_functions.f90
+!! @brief Contains module m_kernel_functions
+
+#:include 'macros.fpp'
+
+!> @brief This module contains kernel functions used to map the effect of the lagrangian bubbles
+!!        in the Eulerian framework.
+module m_kernel_functions
```

Comment:
It might be worthwhile to add something to this name to indicate it's for Lagrangian bubbles. Maybe `m_lag_bub_kernel_functions` or something along those lines?

Reply:
I am updating the name from m_kernel_functions to m_bubbles_EL_kernel_functions, and I have also renamed the lagrangian bubbles module from m_lag_bubbles to m_bubbles_EL, in line with one of Spencer's suggestions.

Reply:
Fixed